Cargo.toml

```diff
@@ -124,7 +124,7 @@ base64 = "0.22"
 bytes = "1"
 chrono = "0.4"
 clap = "3.2"
-conformance-tests = { git = "https://github.com/fermyon/conformance-tests", rev = "ecd22a45bcc5c775a56c67689a89aa4005866ac0" }
+conformance-tests = { git = "https://github.com/itowlson/conformance-tests", rev = "fa529a698616116beb024881cfd6fdeda7f39415" }
```

Comment:
Once conformance test changes are merged, and a new `tests.tar.gz` generated, we can reset to the main fork; at that point, Postgres tests should run.  (They can't run without a merge because they use test definitions from the  `tests.tar.gz` of the conformance-tests repo canary release.)